packages/block-editor/src/components/media-placeholder/README.md

```diff
@@ -111,13 +119,19 @@ The call back receives an array with the new files. Each element of the collecti
 
 - Type: `Function`
 - Required: Yes
+- Platform: Web | Mobile
+
+The argument of the callback is an object containing the following properties:
+- Web: `{ url, alt, id, link, caption, sizes, media_details }`
+- Mobile: `{ mediaId, mediaUrl }`
```

Comment:
I think this should be: 
 Mobile: `{ id, url }`

Reply:
I just pushed it 👍